checkita-core/src/main/scala/org/checkita/dqf/config/jobconf/Sources.scala

```diff
@@ -22,6 +22,7 @@ object Sources {
   sealed abstract class SourceConfig extends JobConfigEntity {
     val keyFields: Seq[NonEmptyString]
     val streamable: Boolean
+    val persist: Option[StorageLevel]
```

Comment:
As `persist` now common to all source configurations, then remove this field from `VirtualSourceConfig` abstract class (line 376). Also, update doc string for SourceConfig to describe presence of `persist` field.